src/components/tab-bar/tab-bar.tsx

```diff
@@ -0,0 +1,139 @@
+    tag: 'limel-tab-bar',
+    styleUrl: 'tab-bar.scss',
+    shadow: true
+})
+export class TabBar {
+
+    @Element()
+    private element: HTMLElement;
+
+    /**
+     * List of tabs to display
+     */
+    @Prop()
+    public tabs: Tab[];
+
+    private mdcTabBar: MDCTabBar;
+
+    @State()
+    private selectedTab = this.tabs[0];
+
+    constructor() {
+        this.handleAction = this.handleAction.bind(this);
+
+    }
+
+    public componentDidLoad() {
+        this.element.focus();
+        this.setup();
+    }
+
+    private setup() {
+        this.element.focus();
+        const element = this.element.shadowRoot.querySelector('.mdc-tab-bar');
+        if (!element) {
+            return;
+        }
+
+        this.mdcTabBar = new MDCTabBar(element);
+        (this.mdcTabBar as any).foundation_.adapter_.getFocusedTabIndex = () => {
+            var tabElements = this.getTabElements();
+            var activeElement = this.element.shadowRoot.activeElement;
+            return tabElements.indexOf(activeElement);
+        };
+
+        this.setupListeners();
+    }
+
+    private getTabElements() {
+        return [].slice.call(this.element.shadowRoot.querySelectorAll('.mdc-tab'));
+    }
+
+    private setupListeners() {
+        this.mdcTabBar.listen(TAB_ACTIVATED_EVENT, this.handleAction);
+    }
+
+    private handleAction(event: MDCTabBarActivatedEvent) {
+        const index = event.detail.index;
+        const selectedTab = this.tabs[index];
+        this.selectedTab = selectedTab;
+    }
+
+    render() {
+        return (
+            <div class="mdc-tab-bar" role="tablist">
+                <div class="mdc-tab-scroller">
+                    <div class="mdc-tab-scroller__scroll-area mdc-tab-scroller__scroll-area--scroll">
+                        <div class="mdc-tab-scroller__scroll-content">
+                            {this.renderTabs()}
+                        </div>
+                    </div>
+                </div>
+
+                <div class="content">
+                    <slot>{this.selectedTab && this.selectedTab.content}</slot>
+                </div>
+            </div>
+        );
+    }
+
+    private renderIcon(tab: Tab) {
+        const style = {};
+        if (tab.iconColor) {
+            style['background-color'] = tab.iconColor;
+            style['color'] = 'white';
+        }
+
+        return (
+            <limel-icon
+                class="mdc-text-field__icon"
+                name={tab.icon}
+                badge={true}
+                style={style}
+                size="small"
+            />
+        )
+    }
+
+    private renderTabs() {
+        return (
+            this.tabs.map((tab, index) => {
+                return (
+                    <button class={`
+                        mdc-tab
+                        ${tab.selected ? 'mdc-tab--active' : ''}
+                    `}
+                        role="tab"
+                        aria-selected="true" tabindex={index}
```

Comment:
I know the code isn't done yet, but I'll comment now or I'll forget to check later:

This isn't right. `aria-selected` should only be true if this is the selected tab, and `tabindex` should be set to `-1` for all but the selected tab. On the selected tab, no tabindex should be set, if I read the docs correctly.